src/character.cpp

```diff
@@ -2655,7 +2655,7 @@ int Character::get_mod_stat_from_bionic( const character_stat &Stat ) const
     return ret;
 }
 
-int Character::get_standard_stamina_cost( item *thrown_item )
+int Character::get_standard_stamina_cost( const item *thrown_item ) const
```

Comment:
These changes should pulled out into a separate cleanup PR.

Reply:
this is a single line of change, i missed it in my other PR that pulled out the plumbing bits, but i'll leave it in this PR unless we're scraping to make it smaller